src/tests/unit/tests/reports/components/report-sections/report-body.test.tsx

```diff
@@ -18,6 +19,16 @@ describe('ReportBody', () => {
         const getScriptStub = () => '';
         const getGuidanceTagsStub = () => [];
         const fixInstructionProcessorMock = Mock.ofType(FixInstructionProcessor);
+        const environmentInfoProviderMock = Mock.ofType(EnvironmentInfoProvider);
+        const environmentInfo: EnvironmentInfo = {
```

Comment:
I'm not sure this is needed on this test. All `ReportBody` is doing with this props is passing them down to the appropriate child components. There is not a call to `environmentInfoProvider.getEnvironmentInfo()` in this class. Can you try remove this and the mock setup and see if that still work without other changes?